docs/src/function.md

```diff
@@ -0,0 +1,8 @@
+# LightGBM.jl
```

Comment:
This file should probably be renamed to "Functions" or "API"  based on how it appears in the generated pages site

Also does it make sense for the header to be LightGBM.jl or something else?

Reply:
Rename the file "Functions".
Also modify the header to a "Functions".